test/util/unit_test.cpp

```diff
@@ -1050,14 +1050,19 @@ void SimpleReporter::begin(const TestContext& context)
     logger.info(format, details.file_name, details.line_number, details.test_name, context.recurrence_index + 1);
 }
 
-void SimpleReporter::fail(const TestContext& context, const char* file_name, long line_number,
-                          const std::string& message)
+void SimpleReporter::fail(const TestContext& context, const char* file, long line, const std::string& message)
 {
     const TestDetails& details = context.test_details;
     util::Logger& logger = context.thread_context.report_logger;
     auto format = context.thread_context.shared_context.num_recurrences == 1 ? "%1:%2: ERROR in %3: %5"
                                                                              : "%1:%2: ERROR in %3#%4: %5";
-    logger.info(format, file_name, line_number, details.test_name, context.recurrence_index + 1, message);
+    auto msg = util::format(format, file, line, details.test_name, context.recurrence_index + 1, message);
```

Comment:
Great idea! Would it be too confusing to log the failure here as well as in the summary at the bottom? I am thinking of the case where a test fails a check and then segfaults.

Reply:
If you insist :-)